src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst

```diff
@@ -0,0 +1,256 @@
+  :end-before: <!-- SPHINX_TemperatureCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_TemperatureTimeHistory -->
+  :end-before: <!-- SPHINX_TemperatureTimeHistoryEnd -->
+
+Similarly, following blocks are needed to collect the solid stress:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressCollection -->
+  :end-before: <!-- SPHINX_SolidStressCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressTimeHistory -->
+  :end-before: <!-- SPHINX_SolidStressTimeHistoryEnd -->
+
+The displacement field can be collected for the whole domain using ``nodeManager`` as follows
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementCollection -->
+  :end-before: <!-- SPHINX_DisplacementCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementTimeHistory -->
+  :end-before: <!-- SPHINX_DisplacementTimeHistoryEnd -->
+
+Also, periodic events are required to allow collecting these data. For example, periodic events for collecting the displacement field are defined as:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementPeriodicEvent -->
+  :end-before: <!-- SPHINX_DisplacementPeriodicEventEnd -->
+
+
+---------------------------------
+Results and benchmark
+---------------------------------
+
+A good agreement between the GEOSX results and analytical results for temperature distribution around the cased wellbore is shown in the figures below:
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/CasedThermoElasticWellbore/thermoElastic_casedWellbore_temperature.py
+
+.. _resultsCasedThermoElasticWellbore_temperatureFig:
+.. figure:: temperature.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Temperature distribution around the cased wellbore
+
+and the validation for the radial displacement around the cased wellbore is shown below:
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/CasedThermoElasticWellbore/thermoElastic_casedWellbore_displacement.py
+
+.. _resultsCasedThermoElasticWellbore_displacementFig:
+.. figure:: displacement.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Radial displacement around the cased wellbore
+
+The validations of the total radial and hoop stress components computed by GEOSX againt reference results are shown in the figure below:
```

Comment:
```suggestion
The validations of the total radial and hoop stress components computed by GEOSX against reference results are shown in the figure below:
```
What is hoop stress?

Reply:
Hoop stress is the tangent stress, this is now clarified in the doc.